### PR TITLE
perf: improve jump link for home page

### PR DESCRIPTION
**Pull request by zouhangwithsweet**
- [x] improve jump link for home page

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ module.exports = {
       appId: '7DCTSU0WBW',
     },
     features,
+    langs: ['en-US', 'zh-CN', 'es', 'fr-FR', 'jp'],
```
Comment (JeremyWuuuuu) [inline docs/.vitepress/config.js:87]: Can we not using hard coded? see the script I wrote in [crowdin-generate.ts](https://github.com/element-plus/element-plus/blob/dev/build/crowdin-generate.ts#L23) for getting it flexible.

Comment (zouhangwithsweet) [reply docs/.vitepress/config.js:87]: Yes, but not necessary.

**Review by sxzz — APPROVED** [review]

File: docs/.vitepress/vitepress/components/vp-navbar.vue [hunk 2]
```diff
@@ -17,13 +18,22 @@ defineEmits(['toggle'])
 const themeEnabled = useFeatureFlag('theme')
 
 const { theme } = useData()
+
+const currentLink = computed(() => {
+  if (!inBrowser) return '/'
+  const exsitLangIndex = theme.value.langs.findIndex((lang) =>
```
Comment (sxzz) [inline docs/.vitepress/vitepress/components/vp-navbar.vue:24]: Maybe a typo `exsitLangIndex`

Comment (JeremyWuuuuu) [reply docs/.vitepress/vitepress/components/vp-navbar.vue:24]:
```suggestion
  const existLangIndex = theme.value.langs.findIndex((lang) =>
```

File: docs/.vitepress/vitepress/components/vp-navbar.vue [hunk 3]
```diff
@@ -17,13 +18,22 @@ defineEmits(['toggle'])
...
+    window?.location?.pathname.startsWith(`/${lang}`)
+  )
+
+  return exsitLangIndex === -1 ? '/' : `/${theme.value.langs[exsitLangIndex]}/`
```
Comment (sxzz) [inline docs/.vitepress/vitepress/components/vp-navbar.vue:28]:
```suggestion
  return existLangIndex === -1 ? '/' : `/${theme.value.langs[exsitLangIndex]}/`
```

**Pull request merged**